src/main/java/unbanner/StudentRepository.java

```diff
@@ -8,8 +8,10 @@
 
 public interface StudentRepository extends MongoRepository<Student, String> {
 
-  public Student findByFirstName(String firstName);
+  public List<Student> findByFirstName(String firstName);
```

Comment:
findByFirstName should not be returning a list

Reply:
only one student has the same first name? 
if we search by first name, it only returns one.
university has many students with same first name

Reply:
on second thought, you are correct about this Mahesh. It absolutely should return a list of students.